src/Controller.vala

```diff
@@ -47,7 +47,7 @@ namespace Vocal {
 
         public bool first_run = true;
         public bool newly_launched = true;
-        public bool should_quit_immediately = true;
```

Comment:
Does this change fix a bug, or is it just flipping the default?

Reply:
This fixes a bug, when this is true by default the setting "Keep playing in background" gets ignored.